### ISSUE TITLE
Failed installing Omnisharp

### ISSUE THREAD

**Issue by PatrickLM** [OmniSharp/omnisharp-vscode#2892]
Starting OmniSharp server at 3/7/2019, 9:33:20 AM
    Target: c:\Users\****\Desktop\Projects\Demo

OmniSharp server started.
    Path: C:\Users\****\.vscode\extensions\ms-vscode.csharp-1.17.1\.omnisharp\1.32.8\OmniSharp.exe
    PID: 11908

The system cannot find the path specified.
[ERROR] Error: OmniSharp server load timed out. Use the 'omnisharp.projectLoadTimeout' setting to override the default delay (one minute).

Uninstalling and installing extensions or VSCode does not work

**Comment by akshita31** [comment]
@PatrickLM Do you have any C# log ? If yes please share the output from that. It seems like the system is not able to install omnisharp? Do you have any proxy enabled that is forbidding the download

**Comment by PatrickLM** [comment]
Resolved:
1) I disabled Kaspersky Firewall
VS-Code immediately updated from 1.31.1 to 1.32.1
2) I re-installed the Omnisharp extension

All other issues got resolved.

Installing C# dependencies...
Platform: win32, x86_64

Downloading package 'OmniSharp for Windows (.NET 4.6 / x64)' (31021 KB).................... Done!
Installing package 'OmniSharp for Windows (.NET 4.6 / x64)'

Downloading package '.NET Core Debugger (Windows / x64)' (43046 KB).................... Done!
Installing package '.NET Core Debugger (Windows / x64)'

Downloading package 'Razor Language Server (Windows / x64)' (46894 KB).................... Done!
Installing package 'Razor Language Server (Windows / x64)'

Finished
